Configuration/Shared/Workspace-Universal-Framework.xcconfig

```diff
@@ -3,6 +3,9 @@
 
 SUPPORTED_PLATFORMS = iphoneos iphonesimulator appletvsimulator appletvos watchsimulator watchos macosx
 
+// Set default SDKROOT
+SDKROOT = iphoneos
```

Comment:
I'm slightly concerned this may eff things up if someone's using this in a `macosx` app - can you point me to the docs that talk about the default SDKRoot and how it affects catalyst stuff?

Reply:
Unfortunately, there's currently not much documentation on how the SDKROOT relates to Catalyst being available or not. However, I think I have found a better way. I have reverted setting the `SDKROOT` and added the `SUPPORTS_MACCATALYST = YES` configuration. That seems a much better solution.